### PR TITLE
feat-route

### PR DESCRIPTION
**Pull request by iamsaurav28**

### CODE REVIEW

File: e-comm-jsx/src/App.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```
Comment (therajatg) [inline e-comm-jsx/src/App.css]: you can remove styles from App.css which are of no use.

File: e-comm-jsx/src/App.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+import React from "react";
+import Header from "./components/front/Header/Header";
+import Routes from "./components/front/Routes/Routes";
+import { BrowserRouter as Router } from "react-router-dom";
```
Comment (therajatg) [inline e-comm-jsx/src/App.js]: import { BrowserRouter as Router } from "react-router-dom";

This should be in index.js

File: e-comm-jsx/src/App.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    <Router>
+    <Header />
+    <Routes />
+    <Footer />
+    </Router>
+   
+  </div>
+  );
+}
+
```
Comment (therajatg) [inline e-comm-jsx/src/App.js:24]: Structure not correct. Import {Routes, Route} from 'react-router-dom' and do this:
<Routes>
<Route path="/" element={<Home/>}>
</Routes>

File: e-comm-jsx/README.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+
+### Deployment
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+
+### `npm run build` fails to minify
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
```
Comment (therajatg) [inline e-comm-jsx/README.md:70]: I like your read me section

**Review by therajatg — APPROVED** [review]

File: e-comm-jsx/src/App.css (hunk 1 above)
Comment (iamsaurav28) [reply e-comm-jsx/src/App.css]: okkk done

File: e-comm-jsx/src/App.js (hunk 2 above)
Comment (iamsaurav28) [reply e-comm-jsx/src/App.js]: done

File: e-comm-jsx/src/App.js (hunk 3 above)
Comment (iamsaurav28) [reply e-comm-jsx/src/App.js:24]: ok

File: e-comm-jsx/README.md (hunk 4 above)
Comment (iamsaurav28) [reply e-comm-jsx/README.md:70]: ty

File: e-comm-jsx/src/components/front/Routes/Routes.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+                 <Wishlist/>
+            </Route>
+            
+            <Route path="/cart" exact>
+                 <Cart/>
+            </Route>
+            <Route path="/auth" exact>
+                 <Auth/>
+            </Route>
+       </Switch>
```
Comment (kjais1720) [inline e-comm-jsx/src/components/front/Routes/Routes.js]: You don't need to use switch anymore, its deprecated, you can directly use `Routes` and `Route` like 
```
<Routes>
    <Route path="/" element={<SomeComponent/>} />
</Routes>
```

File: e-comm-jsx/src/components/front/Header/Header.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+                   <li>
+                        <Link to="/wishlist">Wishlist
+                        </Link>
+                   </li>
+              </ul>
+
+              <ul>
+                   <li>
+                        <Link to="/cart">
+                             <i className='fa fa-shopping-cart'/>
```
Comment (kjais1720) [inline e-comm-jsx/src/components/front/Header/Header.js]: If you are giving an icon to the cart link, then do the same for the wishlist link to maintain design consistency

File: e-comm-jsx/src/components/front/Routes/Routes.js (hunk 5 above)
Comment (iamsaurav28) [reply e-comm-jsx/src/components/front/Routes/Routes.js]: yes but my file name is also same, so its getting error.

File: e-comm-jsx/src/components/front/Header/Header.js (hunk 6 above)
Comment (iamsaurav28) [reply e-comm-jsx/src/components/front/Header/Header.js]: okk done

File: e-comm-jsx/src/components/front/Routes/Routes.js (hunk 5 above)
Comment (kjais1720) [reply e-comm-jsx/src/components/front/Routes/Routes.js]: change the filename then, it's confusing as well. You can name the file like `AllRoutes` or something else, you should not have two components of the same name

Comment (iamsaurav28) [reply e-comm-jsx/src/components/front/Routes/Routes.js]: ok

**Review by kjais1720 — APPROVED** [review]
Looks good now👍

**Pull request merged**